conf.py

```diff
@@ -103,3 +104,96 @@
                                   "README.html")): # don't create dangling symlinks;
                                                    # it annoyes Github Pages.
                 os.symlink("README.html", z)
+
+
+
+
+# -- Custom scripts ----------------------------------------------------------
+
+import os
+from pathlib import Path
+import random
+import requests
+from subprocess import run
+from textwrap import dedent
+from urllib.parse import urlparse
+from ghapi.all import GhApi
+import pandas as pd
+
+import yaml
+
+from sphinx.application import Sphinx
+from sphinx.util import logging
+
+LOGGER = logging.getLogger("conf")
+
+def build_gallery(app: Sphinx):
+    # Build the gallery file
+    LOGGER.info("building gallery...")
+    grid_items = []
+    projects = yaml.safe_load((Path(app.srcdir) / "gallery.yml").read_text())
+    random.shuffle(projects)
```

Comment:
'shuffle'?

Reply:
ah! thanks, i was wondering why they were shuffled 😅 

fixed in c3e6c8c409a122de911d45f80b6b248865ef05ae